plugins/out_cloudwatch_logs/cloudwatch_logs.c

```diff
@@ -348,50 +347,53 @@ static int cb_cloudwatch_init(struct flb_output_instance *ins,
     flb_output_upstream_set(upstream, ctx->ins);
     ctx->cw_client->host = ctx->endpoint;
 
-    /* alloc the payload/processing buffer */
+    /* Export context */
+    flb_output_set_context(ins, ctx);
+
+    return 0;
+
+error:
+    flb_free(session_name);
+    flb_plg_error(ctx->ins, "Initialization failed");
+    flb_cloudwatch_ctx_destroy(ctx);
+    return -1;
+}
+
+struct cw_flush *new_buffer()
```

Comment:
As you noted yourself to me, need to also remove the singleplex scheduler.

Reply:
The singleplex scheduler has been removed.